stories/Dropdown/examples.jsx

```diff
@@ -100,6 +100,40 @@ CustomLabel.CODE = `<Dropdown
   items={['Lemon', 'Banana', 'Strawberry', 'Orange', 'Avocado']}
 />`;
 
+const CustomLabelWithDescription = () => (
```

Comment:
When the item is clicked the dropdown is not updated with the selectedItem. Can we include that in the example?

Reply:
I did update `Controlled` example because that isn't impossible to use React Hooks or duplicate class on stories files.